lib/Command/Test.php

```diff
@@ -46,9 +46,10 @@ protected function configure() {
 	}
 
 	public function execute(InputInterface $input, OutputInterface $output) {
-		$this->logger->info("staring sentry test command");
+		$this->logger->info("starting sentry test command");
```

Comment:
I prefer `staring`. Makes the app sound like a movie start :P

Reply:
well but then I would have to be "starring" right?

**starring Sentry the test command**